common/src/main/nl/jft/common/glicko/StandardGlickoCalculator.java

```diff
@@ -0,0 +1,316 @@
+package nl.jft.common.glicko;
+
+/**
+ * The {@code StandardGlickoCalculator} is an implementation of the {@code GlickoCalculator}.
+ * It implements the Glicko rating system as described by Mark Glickman.
+ *
+ * The only difference between this implementation and the default implementation is that is does not support evaluating multiple
+ * results at the same time. Every result has to be evaluated after the completion of a match and a new rating has to be assigned.
+ *
+ * @author Oscar de Leeuw
+ */
+public class StandardGlickoCalculator implements GlickoCalculator {
+
+    private static final double DEFAULT_RATING = 1500d;
+    private static final double DEFAULT_DEVIATION = 350d;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "DEFAULT_DEVIATION" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://92.110.24.119:9000/coding_rules#rule_key=squid%3AS1068)